source/reference/minio-mc/mc-ilm-rule-add.rst

```diff
@@ -115,6 +116,16 @@ Parameters
 
    - :mc-cmd:`~mc ilm rule add --expire-delete-marker`
 
+.. mc-cmd:: --expire-all-object-versions
+   :optional:   
+
+   .. versionadded:: mc RELEASE.2024-02-24T01-33-20Z
+
+   Expire all current **and** noncurrent versions of an object.
+   Use with the :mc-cmd:`~mc ilm rule add --expire-days` option to specify the number of days after which all versions of an object should be deleted by the scanner process.
+
+   After the scanner processes this command, no versions of the object remain on the deployment.
```

Comment:
A good place to link to info on the scanner? Something that makes it clear this isn't instantaneous.